### DISCUSSION TITLE
De-privileging Administrator Account

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#3888] [Various ideas and suggestions]
Some software forces to launch itself with full elevation when user is part of administrator group, even when full elevation is not desired. For example, WireGuard for Windows launches itself with administrative rights for any user who is part of administrator group. That results in UI showing too much informaiton (public key, peer key, IP's, ports), but there is also a restricted non-administrative user access for user who are members of "Network Configuration Operator" group, in which case WireGuard for Windows hides sensitive UI elements. 

It would be nice if Sandboxie had some options for green boxes to run processes with user-customized selectable privileges or as member of whichever group.

**Comment by DavidXanatos** [comment]
i created an issue out of that https://github.com/sandboxie-plus/Sandboxie/issues/3929 as feature request
